docs/source/daml/intro/daml/daml-intro-8/daml/Intro/Exceptions.daml

```diff
@@ -155,12 +155,61 @@ test = script do
   -- This transfer happened, so balance is now 900
   ((_, alice_account)::_) <- query @Account alice
   alice_account.amount === 900.0
+  -- and there's an Iou from Alice's bank to the shop
+  [(_, iou_bank)] <- query @Iou bank
+  iou_bank === Iou with issuer = bank, owner = shopOwner, amount = 100.0
+  -- Now let's try to go over the limit
   submit alice $ exerciseCmd shop (OrderItemTrustedLimited alice)
-  -- This one didn't, so balance is still 900
+  -- This one doesn't pass, so balance is still 900
   ((_, alice_account)::_) <- query @Account alice
   alice_account.amount === 900.0
+  -- and the newly created Iou is in Alice's name instead of her bank's
+  [(_, iou_alice)] <- query @Iou alice
+  [(_, iou_bank)] <- query @Iou bank
+  iou_bank === Iou with issuer = bank, owner = shopOwner, amount = 100.0
+  iou_alice === Iou with issuer = alice, owner = shopOwner, amount = 100.0
 
-  pure ()
+test2 = script do
+  bank <- allocateParty "Bank"
+  alice <- allocateParty "Alice"
+  shopOwner <- allocateParty "ShopOwner"
+
+  alice_account_id <-
+    submitMulti [bank, alice] [] $ createCmd Account with
+      issuer = bank
+      owner = alice
+      amount = 1000.0
+
+  submitMulti [alice, shopOwner] [] $
```

Comment:
Theres `submitMultiMustFail`, so you could drop the case from the contract just exercise against the account directly?

Reply:
I thought about it but that would make a weaker test, since the exercise could fail in many different ways

Reply:
Ah fair enough, seems we could do with some kind of assetError in script land for submitting commands

Reply:
indeed, that would be nice to have!